KsApi/models/graphql/adapters/Backing+BackingFragmentTests.swift

```diff
@@ -18,15 +18,15 @@ final class Backing_BackingFragmentTests: XCTestCase {
   func test_noReward() {
     do {
       var dict = backingDictionary()
-      dict["addOns"] = nil
-      dict["reward"] = nil
+      dict["addOns"] = NSNull()
+      dict["reward"] = NSNull()
```

Comment:
Good old `NSNull`. Needed this so that the key isn't removed the dictionary completely. Apollo needs the key and a `nil` value for it to correctly parse a "no reward".

Reply:
Ah very cool, I didn't realize you couldn't add `nil` to dictionaries and arrays in Swift and `NSNull()` is a placeholder that represents no value.

Reply:
Had to read up on it a bit [here](https://stackoverflow.com/questions/836601/whats-the-difference-between-nsnull-null-and-nil/21484587)

Reply:
Yeah it was/is commonly used in Objective-C.